CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+#Code owners are automatically requested for review when someone opens a pull request that modifies code that they own.
```

Comment:
nit: space between the `#`  and the first character.

Reply:
fixed